src/renderer/chat/ChatTab.ts

```diff
@@ -246,6 +247,17 @@ export default class ChatTab {
       if (this.textAreaElt.value.length > 0) return;
       this.textAreaElt.value = this.previousMessage;
       event.preventDefault();
+    }else if(event.keyCode == 9 /* TAB */){        

```

Comment:
Should be `} else if (event.keyCode === 9 /* Tab */) {`

 * Missing spaces
 * Should use triple equals
 * Tab shouldn't be all caps
 * Removed trailing spaces at the end of line

If you code using VS Code, I recommend installing the `tslint` extension, it'll notify you for most of these issues.